test/e2e/nodeportlocal_test.go

```diff
@@ -73,15 +78,15 @@ func configureNPLForAgent(t *testing.T, data *TestData, startPort, endPort int)
 // TestNodePortLocal is the top-level test which contains all subtests for
 // NodePortLocal related test cases so they can share setup, teardown.
 func TestNodePortLocal(t *testing.T) {
-	skipIfNotIPv4Cluster(t)
-	skipIfNodePortLocalDisabled(t)
-
 	data, err := setupTest(t)
 	if err != nil {
 		t.Fatalf("Error when setting up test: %v", err)
 	}
 	defer teardownTest(t, data)
 
+	skipIfNotIPv4Cluster(t)
```

Comment:
I think you can keep this line as before so it can skip earlier.

Reply:
Moved `skipIfNotIPv4Cluster`  to the front of the data part.